website/content/zh/wgs/platforms/whitepaper/index.md

```diff
@@ -2,504 +2,219 @@
+
+![产品、平台和能力提供者之间的关系](platform_components.png)
+
+我们在本文中着重讨论了如何构建良好的平台和平台团队；现在在最后一节中，我们将描述平台实际上可能提供的能力。这个列表旨在指导平台构建者，并包括典型的云原生应用程序所需的功能。正如我们一直在强调的，良好的平台反映了其用户的需求，因此最终平台团队应该与其用户一起选择和优先考虑平台所提供的功能。
+
+能力可以包括几个*特性*，意味着父能力域的方面或属性。例如，可观测性可能包括用于收集和发布度量、跟踪和日志以及观察成本和能源消耗的特性。在您的组织中考虑每个特性或方面的需求和优先级。随后的 CNCF 出版物可能会进一步扩展每个域。
+
+在构建云原生计算平台时要考虑以下能力领域：
+
+1. 用于观察和配置产品和能力的 **Web 门户**
+2. 用于自动配置产品和能力的 **APIs**（和 CLIs）
+3. **“黄金路径”模板和文档**，以便在产品中实现最佳使用
+4. 用于构建和测试服务和产品的**自动化**
+5. 用于交付和验证服务和产品的**自动化**
+6. **开发环境**，如托管的 IDE 和远程连接工具
+7. 使用仪器和仪表板的服务和产品的**可观测性**，包括功能、性能和成本的观察
+8. **基础设施**服务，包括计算运行时、可编程网络以及块和卷存储
+9. **数据**服务，包括数据库、缓存和对象存储
+10. 包括代理、队列和事件织物的**消息**和事件服务
+11. **身份和密码**管理服务，例如服务和用户身份和授权、证书和密钥发行以及静态密码存储
+12. 包括代码和工件的静态分析、运行时分析和策略执行的**安全**服务
+13. 包括容器镜像和特定语言包、自定义二进制文件和库以及源代码的**工件存储**
+
+以下表格旨在帮助读者通过松散相关现有 CNCF 或 CDF 项目来理解每种能力。
+
+| 能力                          | 描述                                                         | 示例 CNCF/CDF 项目                                           |
+| :---------------------------- | :----------------------------------------------------------- | :----------------------------------------------------------- |
+| 用于配置和观察能力的 Web 门户 | 发布文档、服务目录和项目模板。发布有关系统和能力的遥测数据。 | Backstage、Skooner、Ortelius                                 |
+| 自动配置能力的 API            | 用于自动创建、更新、删除和观察能力的结构化格式。             | Kubernetes、Crossplane、Operator Framework、Helm、KubeVela   |
+| 黄金路径模板和文档            | 用于快速项目开发的良好集成代码和能力的模板化组合。           | ArtifactHub                                                  |
+| 用于构建和测试产品的自动化    | 自动构建和测试数字产品和服务。                               | Tekton、Jenkins、Buildpacks、ko、Carvel                      |
+| 用于交付和验证服务的自动化    | 自动化和观察服务的交付。                                     | Argo、Flux、Keptn、Flagger、OpenFeature                      |
+| 开发环境                      | 启用应用程序和系统的研究和开发。                             | Devfile、Nocalhost、Telepresence、DevSpace                   |
+| 应用程序可观测性              | 仪表应用程序，收集和分析遥测数据，并将信息发布给利益相关者。 | OpenTelemetry、Jaeger、Prometheus、Thanos、Fluentd、Grafana、OpenCost |
+| 基础设施服务                  | 运行应用程序代码、连接应用程序组件并为应用程序持久化数据     | Kubernetes、Kubevirt、Knative、WasmEdgeCNI、Istio、Cilium、Envoy、Linkerd、CoreDNSRook、Longhorn、Etcd |
+| 数据服务                      | 持久化应用程序的结构化数据                                   | TiKV、Vitess、SchemaHero                                     |
+| 消息和事件服务                | 使应用程序异步通信                                           | Strimzi、NATS、gRPC、Knative、Dapr                           |
+| 身份和密码服务                | 确保工作负载具有定位器和密码来使用资源和能力。使服务能够向其他服务识别自己 | Dex、External Secrets、SPIFFE/SPIRE、Teller、cert-manager    |
+| 安全服务                      | 观察运行时行为并报告 / 纠正异常。验证构建和工件不包含漏洞。根据企业要求限制平台上的活动；通知和 / 或纠正异常 | Falco、In-toto、KubeArmor、OPA、Kyverno、Cloud Custodian     |
+| 工件存储                      | 存储、发布和保护用于生产的内置工件。缓存和分析第三方工件。存储源代码。 | ArtifactHub、Harbor、Distribution、Porter                    |
+
+## 术语表
```

Comment:
Great job.  👍 

I think it might be a version difference due to time difference, the version you translated at that time already has some inconsistencies with the current English version, for example the glossary part has migrated out in the current version. Because of that, I would suggest checking and updating this section to be consistent with the current stable version would be better.

Reply:
v1.0 paltform white paper  https://github.com/cncf/tag-app-delivery/blob/main/platforms-whitepaper/v1/index.md

Reply:
I didn't notice any content changes. I'm using plain Markdown format without embedded HTML-format tables like in the original text. Can you point out specific examples of content changes?

Reply:
We should use the latest version  under en/wgs/ instead of zh/wgs/ to translate 😄

Reply:
Updated